rdkit/Chem/SaltRemover.py

```diff
@@ -33,20 +33,61 @@
 
 import os
 import re
+from rdkit.Chem.rdmolfiles import SDMolSupplier
 
 from rdkit import Chem
 from rdkit import RDConfig
 from contextlib import closing
 
+class Format:
+  Smarts, Mol, Smiles = range(3)
```

Comment:
May I use it? I thought `enum` is part of **Python 3** but I should keep with **Python 2**. Then; I would need to add an external dependency and I thought this should be avoided.